app/angular/package.json

```diff
@@ -51,6 +50,10 @@
     "@angular/forms": ">=6.0.0",
     "@angular/platform-browser": ">=6.0.0",
     "@angular/platform-browser-dynamic": ">=6.0.0",
-    "babel-loader": "^7.0.0 || ^8.0.0"
+    "babel-loader": "^7.0.0 || ^8.0.0",
+    "zone.js": "^0.8.26"
+  },
+  "devDependencies": {
+    "zone.js": "^0.8.26"
```

Comment:
@kroeder should we add zone.js in cli?

Reply:
You mean when running `sb init`? No, there's no angular without zone.js installed. `ng new my-app` already does add zone.js, angular does not work without it - We do not need to take care of this

https://medium.com/@MertzAlertz/what-the-hell-is-zone-js-and-why-is-it-in-my-angular-2-6ff28bcf943e

> Let’s do an experiment. Go and take your favorite Angular 2 app, and don’t include zone.js. See what happens, I’ll wait here. Notice anything interesting? None of the bindings are working!